src/tokens.js

```diff
@@ -386,12 +390,12 @@ const tokens = {
    *
    * @throws {InsufficientFundsError} If not enough tokens for deposit
    *
-   * @return {Object} Mint data {'inputs', 'outputs', 'tokens'}
+   * @return {Promise} Promise resolved with object containing mint data {'inputs', 'outputs', 'tokens'}
    *
    * @memberof Tokens
    * @inner
    */
-  createMintData(mintInput, token, address, amount, depositInputs, options) {
+  async createMintData(mintInput, token, address, amount, depositInputs, options) {
```

Comment:
It looks weird to me that we're mixing `async/await` with promises, especially in the same function.